examples/elastic/lamp-basic.yml

```diff
@@ -1,126 +1,149 @@
 ROSTemplateFormatVersion: '2015-09-01'
 Description:
-  en: On the existing virtual proprietary network, switches, and security group base
-    resources, the one-click deployment OF LAMP (Linux-Apache-MySQL-PHP) development
-    environment based on the Centos7 system.
+  en: On the existing virtual proprietary network, switches, and security group base resources, the one-click deployment OF LAMP (Linux-Apache-MySQL-PHP) development environment based on the Centos7 system.
   zh-cn: 在已有虚拟专有网络、交换机和安全组基础资源上，基于Centos7系统中一键部署LAMP(Linux+Apache+MySQL+PHP)开发环境。
 Parameters:
+  SelectExistECS:
+    Type: Boolean
+    Label:
+      en: Whether to select an existing instance
+      zh-cn: 是否选择已有实例
+    Description:
+      en: Whether to select an existing Centos7 system instance. The Centos7 system instance must be connected to a public network.
+      zh-cn: 是否选择已有Centos7系统实例，实例需要开通公网。
+    Default: false
   VPC:
     Type: String
     Label:
       en: Existing VPC Instance ID
       zh-cn: 现有VPC的实例ID
     Description:
-      en: Please search the ID starting with (vpc-xxx)from console-Virtual Private
-        Cloud
+      en: Please search the ID starting with (vpc-xxx)from console-Virtual Private Cloud
```

Comment:
Please add a period at the end of the sentence.

Please also check other parameter descriptions.